docroot/themes/humsci/humsci_basic/humsci_basic.theme

```diff
@@ -295,3 +295,11 @@ function humsci_basic_preprocess_paragraph(&$variables) {
     }
   }
 }
+
+/**
```

Comment:
This preprocess is not necessary, please remove it. I initially thought that the `data-` attribute was going to be applied on the `<figure>`, but it's applied on the `<article>` that contains the media entity. That element already has a class that indicates the type (`hb-media-image`, `hb-media-video`, etc.)